R/methods-AbundanceData.R

```diff
@@ -67,8 +77,18 @@ setMethod("getSampleMetadata", signature("AbundanceData"), function(object, asCo
     dt <- data.table::copy(dt)
   }
 
+  if (object@removeEmptySamples) {
+    # not using getAbundances here bc i want the empty samples here
```

Comment:
why not make a findEmptySamples function that returns the sample names? We could use that in this fn, the get abundances fn, and if we wanted to print out the names of empty samples elsewhere.

Reply:
im a little nervous at the idea of relying on/ interacting w the id cols too much. im not sure i want to be doing things like validating that we have unique ids etc here..